gulp/tasks/copy.js

```diff
@@ -36,10 +38,30 @@ gulp.task( 'copy:vendorjs', function() {
     } ) );
 } );
 
+gulp.task( 'copy:usage', function() {
+  return gulp.src( config.usage.src )
+    .pipe( $.changed( config.usage.dest ) )
+    .pipe( $.tap( function ( file, t ) {
+      fileName = path.dirname( file.path )
+        .split( path.sep )
+        .pop();
+    } ) )
+    .pipe( $.rename( function ( path ) {
+      path.basename = fileName;
+      path.extname = '.md'
+    } ) )
```

Comment:
What's all this about? Isn't this PR proposing just copying straight `*.md` to `*.md` files?